### PR TITLE
AMLS 4733 | Removed showFees toggle

### PR DESCRIPTION
**Pull request by owen-opencast**
Removed showFees toggle and updated spec tests as appropriate.

## How Has This Been Tested?

Passed spec tests and acceptance tests

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/utils/DeclarationHelper.scala [hunk 1]
```diff
@@ -54,16 +54,15 @@ object DeclarationHelper {
...
+  def routeDependingOnNominatedOfficer(hasNominatedOfficer: Boolean, status: SubmissionStatus) = {
     hasNominatedOfficer match {
-      case true => routeWithNominatedOfficer(status, showFees)
+      case true => routeWithNominatedOfficer(status)
       case false => routeWithoutNominatedOfficer(status)
     }
   }
 
-  private def routeWithNominatedOfficer(status: SubmissionStatus, showFees: Boolean) = {
+  private def routeWithNominatedOfficer(status: SubmissionStatus) = {
```
Comment (slavvco) [inline app/utils/DeclarationHelper.scala]: can be refactored

File: app/controllers/declaration/WhoIsTheBusinessNominatedOfficerController.scala [hunk 2]
```diff
@@ -96,7 +96,6 @@ class WhoIsTheBusinessNominatedOfficerController @Inject ()(
               rp <- updateNominatedOfficer(responsiblePeople, data)
               _ <- dataCacheConnector.save(ResponsiblePerson.key, rp)
             } yield serviceStatus match {
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/declaration/WhoIsTheBusinessNominatedOfficerController.scala]: Removed

File: app/utils/DeclarationHelper.scala (hunk 1 above)
Comment (owen-opencast) [reply app/utils/DeclarationHelper.scala]: Done

File: app/controllers/FeeGuidanceController.scala [hunk 3]
```diff
@@ -40,14 +38,13 @@ class FeeGuidanceController @Inject()(val authConnector: AuthConnector,
                                       val dataCacheConnector: DataCacheConnector,
                                       appConfig: AppConfig) extends BaseController {
 
-  def get = FeatureToggle(appConfig.showFeesToggle) {
+  def get =
```
[the remark this answers is not in the document]
Comment (owen-opencast) [reply app/controllers/FeeGuidanceController.scala]: Removed FeeGuidanceController

File: app/controllers/responsiblepeople/FitAndProperController.scala [hunk 4]
```diff
@@ -49,10 +49,9 @@ class FitAndProperController @Inject()(
         case Some(ResponsiblePerson(Some(personName),_,_,_,_,_,_,_,_,_,_,_,_,_,_,alreadyPassed,_,_,_,_,_,_))
           if alreadyPassed.hasAlreadyPassedFitAndProper.isDefined =>
           Ok(views.html.responsiblepeople.fit_and_proper(Form2[Boolean](alreadyPassed.hasAlreadyPassedFitAndProper.get),
-            edit, index, flow, personName.titleName, appConfig.showFeesToggle, appConfig.phase2ChangesToggle))
```
Comment (ajchilts) [inline app/controllers/responsiblepeople/FitAndProperController.scala]: @owen-opencast - did you intend to remove 'appConfig.phase2ChangesToggle'?

File: app/controllers/responsiblepeople/FitAndProperController.scala [hunk 5]
```diff
@@ -66,7 +65,7 @@ class FitAndProperController @Inject()(
             case f: InvalidForm =>
               getData[ResponsiblePerson](index) map { rp =>
                 BadRequest(views.html.responsiblepeople.fit_and_proper(f, edit, index, flow,
-                  ControllerHelper.rpTitleName(rp), appConfig.showFeesToggle, appConfig.phase2ChangesToggle))
```
Comment (ajchilts) [inline app/controllers/responsiblepeople/FitAndProperController.scala]: @owen-opencast - did you intend to remove 'appConfig.phase2ChangesToggle'?

File: app/controllers/responsiblepeople/FitAndProperNoticeController.scala [hunk 6]
```diff
@@ -16,31 +16,23 @@
...
                                         val dataCacheConnector: DataCacheConnector,
-                                        val authConnector: AuthConnector,
-                                        appConfig: AppConfig
+                                        val authConnector: AuthConnector
                                       ) extends RepeatingSection with BaseController {
 
   def get(index: Int, edit: Boolean = false, flow: Option[String] = None) = Authorised.async {
     implicit authContext => implicit request =>
-        Future(Ok(views.html.responsiblepeople.fit_and_proper_notice(EmptyForm, edit, index, flow, "",
-            appConfig.showFeesToggle, appConfig.phase2ChangesToggle)))
```
Comment (ajchilts) [inline app/controllers/responsiblepeople/FitAndProperNoticeController.scala]: @owen-opencast - did you intend to remove 'appConfig.phase2ChangesToggle'?

File: app/views/responsiblepeople/fit_and_proper.scala.html [hunk 7]
```diff
@@ -18,7 +18,7 @@
 @import include._
 @import include.forms2._
 
-@(f: Form2[_], edit: Boolean, index: Int, flow: Option[String], personName: String, showFees: Boolean, phase2Toggle: Boolean)(implicit request: Request[_],m:Messages, lang: Lang)
```
Comment (ajchilts) [inline app/views/responsiblepeople/fit_and_proper.scala.html]: @owen-opencast - did you intend to remove 'appConfig.phase2ChangesToggle'?

File: app/views/responsiblepeople/fit_and_proper_notice.scala.html [hunk 8]
```diff
@@ -18,7 +18,7 @@
 @import include._
 @import include.forms2._
 
-@(f: Form2[_], edit: Boolean, index: Int, flow: Option[String], personName: String, showFees: Boolean, phase2Toggle: Boolean)(implicit request: Request[_],m:Messages, lang: Lang)
```
Comment (ajchilts) [inline app/views/responsiblepeople/fit_and_proper_notice.scala.html]: @owen-opencast - did you intend to remove 'appConfig.phase2ChangesToggle'?

**Pull request merged**